### ISSUE TITLE
Continuation of userplugins repository

### ISSUE THREAD

**Issue by RobinRMC** [D3SOX/Vencord#23]
### Content

Regarding [D3SOX/vencord-userplugins](https://github.com/D3SOX/vencord-userplugins):

It's still possible to have multiple userplugins in one repository. [Kyuuhachi/VencordPlugins](https://github.com/Kyuuhachi/VencordPlugins) still works fine.

Could you look into this?

### Request Agreement

- [X] I have read the requirements for opening an issue above

**Comment by D3SOX** [comment]
Yeah, my wording wasn't that great. It's not possible to have native code in a repo with multiple plugins anymore (or at least I didn't make it) and I don't want to provide them without an updater

**Comment by RobinRMC** [comment]
But why is an updater needed? People can just close the repository, right? Just like they do with any other userplugin. It's not that hard, and if you can build/compile Vencord, you can clone a repository.

**Comment by nnoc** [comment]
> But why is an updater needed? People can just clone the repository, right? Just like they do with any other userplugin. It's not that hard, and if you can build/compile Vencord, you can clone a repository.

I also do not understand the need for an updater for the same reasons, we do pull/update all other user shared plugin repositories manually and build anyways. (just made a bat file)

**Comment by D3SOX** [comment]
I think the best solution for all that allows me to continue to work how I like would be to have separate repos but sync them from the main repo. I will see when I find time to implement that.
